src/lib-components/NavBreadcrumb.vue

```diff
@@ -0,0 +1,86 @@
+<script>
+// SVGs
+import SvgIconCaretLeft from 'ucla-library-design-tokens/assets/svgs/icon-caret-left.svg'
+
+export default {
+  name: 'NavBreadcrumb',
+  components: {
+    SvgIconCaretLeft,
+  },
+  props: {
+    to: {
+      type: String,
+      default: '',
+    },
+    parentTitle: {
+      type: String,
+      default: '',
+    },
+    title: {
+      type: String,
+      default: '',
+    },
+  },
+}
+</script>
+
+<template>
+  <div class="nav-breadcrumb subtitle">
+    <smart-link :to="to" class="parent-page-url" v-text="parentTitle" />
```

Comment:
We need to import smartlink component in this file, I think we never did in production too.